tests/scripts/thread-cert/node.py

```diff
@@ -40,6 +40,9 @@
 import unittest
 import binascii
 
+EXPECT_LINE_START = r'(?<=[\r\n])'
+EXPECT_LINE_END = r'(?=[\r\n])'
```

Comment:
Limit them inside the class or method?

Reply:
They are constants used by multiple methods. If we declare as class member, access them would look like `node._EXPECT_LINE_START`, which looks really weird. So I tend to keep them a module global constants.